[2025-10-26T17:15:09.264+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-26T17:15:09.294+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flood_prediction_pipeline.send_slack_alert scheduled__2025-10-26T17:10:00+00:00 [queued]>
[2025-10-26T17:15:09.306+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flood_prediction_pipeline.send_slack_alert scheduled__2025-10-26T17:10:00+00:00 [queued]>
[2025-10-26T17:15:09.308+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-26T17:15:09.326+0000] {taskinstance.py:2330} INFO - Executing <Task(SlackWebhookOperator): send_slack_alert> on 2025-10-26 17:10:00+00:00
[2025-10-26T17:15:09.338+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=447) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-26T17:15:09.337+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'flood_prediction_pipeline', 'send_slack_alert', 'scheduled__2025-10-26T17:10:00+00:00', '--job-id', '891', '--raw', '--subdir', 'DAGS_FOLDER/flood_prediction_dag.py', '--cfg-path', '/tmp/tmpdt8shh3i']
[2025-10-26T17:15:09.340+0000] {standard_task_runner.py:63} INFO - Started process 450 to run task
[2025-10-26T17:15:09.340+0000] {standard_task_runner.py:91} INFO - Job 891: Subtask send_slack_alert
[2025-10-26T17:15:09.390+0000] {task_command.py:426} INFO - Running <TaskInstance: flood_prediction_pipeline.send_slack_alert scheduled__2025-10-26T17:10:00+00:00 [running]> on host ac7c4fe30011
[2025-10-26T17:15:09.496+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flood_prediction_pipeline' AIRFLOW_CTX_TASK_ID='send_slack_alert' AIRFLOW_CTX_EXECUTION_DATE='2025-10-26T17:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-26T17:10:00+00:00'
[2025-10-26T17:15:09.497+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-26T17:15:09.545+0000] {base.py:84} INFO - Using connection ID 'slack_webhook_default' for task execution.
[2025-10-26T17:15:10.240+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-26T17:15:10.253+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=flood_prediction_pipeline, task_id=send_slack_alert, run_id=scheduled__2025-10-26T17:10:00+00:00, execution_date=20251026T171000, start_date=20251026T171509, end_date=20251026T171510
[2025-10-26T17:15:10.281+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-26T17:15:10.304+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-26T17:15:10.309+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
